contrib/http_servlet/src/test/java/io/opencensus/contrib/http/servlet/OcHttpServletFilterTest.java

```diff
@@ -93,17 +92,16 @@ public void testInit() throws IOException, ServletException {
     when(mockConfig.getServletContext()).thenReturn(mockServletContext);
     when(mockServletContext.getAttribute(OC_TRACE_PROPAGATOR)).thenReturn(null);
     when(mockServletContext.getAttribute(OC_EXTRACTOR)).thenReturn(null);
-    when(mockServletContext.getAttribute(OC_PUBLIC_ENDPOINT)).thenReturn(null);
+    when(mockServletContext.getInitParameter(OC_PUBLIC_ENDPOINT)).thenReturn("false");
```

Comment:
Change "false" to null here and add following line after line 100
verify(mockServletContext).getInitParameter()

This is to verify that getInitParameter() is called once.

Reply:
Changed as suggested in latest push